auto_ria/auto_ria/spiders/ria_spider.py

```diff
@@ -1,14 +1,37 @@
 import scrapy
 import requests
+import re
 from collections import OrderedDict
 from bs4 import BeautifulSoup
 
-class QuotesSpider(scrapy.Spider):
+class RiaSpider(scrapy.Spider):
     name = "auto_ria"
-    search_url = 'https://auto.ria.com/search/?category_id=1&marka_id[0]=84&model_id[0]=31495&s_yers[0]=2003&po_yers[0]=2004&currency=1&abroad=2&custom=1&fuelRatesType=city&engineVolumeFrom=&engineVolumeTo=&power_name=1&countpage=10'
+#    search_url = 'https://auto.ria.com/search/?category_id=1&marka_id[0]=84&model_id[0]=31495&s_yers[0]=2003&po_yers[0]=2004&currency=1&abroad=2&custom=1&fuelRatesType=city&engineVolumeFrom=&engineVolumeTo=&power_name=1&countpage=10'
```

Comment:
If this line is not needed - better to remove it, if needed, maybe add a comment.
Also the starting `#` should be indented, on the `name = "auto_ria"` level

Reply:
Removed :)